cray_product_catalog/catalog_update.py

```diff
@@ -57,6 +57,7 @@
 PRODUCT = os.environ.get("PRODUCT").strip()  # required
 PRODUCT_VERSION = os.environ.get("PRODUCT_VERSION").strip()  # required
 CONFIG_MAP = os.environ.get("CONFIG_MAP", "cray-product-catalog").strip()
+PRODUCT_CONFIG_MAP = CONFIG_MAP + "-" + PRODUCT
```

Comment:
Just a thought for scalability, we can create a tuple of CMs and then use this tuple in `main` inside a loop and create CMs. Also the function `split_catalog_data` shall return a tuple of main,product data. e.g. (main_data, prod_data)
```suggestion
CONFIG_MAP = (CONFIG_MAP, f"{CONFIG_MAP}-{PRODUCT}")
```

Main() will be something like this:

splitted_data = split_catalog_data(data)
for ind, cm in enumerate(CONFIG_MAP):
     update_config_map(splitted_data[ind], cm, CONFIG_MAP_NAMESPACE)